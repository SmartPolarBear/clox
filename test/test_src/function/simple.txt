R"(
fun fuck(first:string, second:string) {
    print "I fucked "+first+" firstly, and then fucked "+second+".";
}

fuck("AA","BB");
)"