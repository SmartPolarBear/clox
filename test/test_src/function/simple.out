R"(I fucked AA firstly, and then fucked BB.)"